app/components/global/layout/media.js

```diff
@@ -0,0 +1,20 @@
+import { css } from 'styled-components'
+import theme from '@elifesciences/elife-theme'
```

Comment:
Note that this breaks the assumption that the theme is configured via the themeprovider only. The way this helper is being used so far is always within a styled component, so it could return a function which then reads the theme off the `props`.